php/class-customize-snapshot-manager.php

```diff
@@ -494,9 +502,6 @@ public function update_snapshot() {
 		if ( ! check_ajax_referer( self::AJAX_ACTION, 'nonce', false ) ) {
 			status_header( 400 );
 			wp_send_json_error( 'bad_nonce' );
-		} elseif ( ! current_user_can( 'customize' ) ) {
```

Comment:
You will never get pass this condition `} elseif ( empty( $this->unsanitized_snapshot_post_data ) ) {` because `current_user_can( 'customize' )` will be false in `Customize_Snapshot_Manager::capture_unsanitized_snapshot_post_data`